smart-contracts/dev-environments/remix/troubleshooting-faq.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Troubleshooting and FAQs
```

Comment:
Remember this is for SEO purposes and is what shows up on search engines, so what's going to differentiate this title on google compared to the Hardhat Troubleshooting page?